### PR TITLE
Improve test coverage

### PR DESCRIPTION
**Pull request by Jasopaum**
## Description
We need 100% test coverage, this is a step towards this goal.

## Affected Dependencies
None

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: src/syft/core/io/location/specific.py [hunk 1]
```diff
@@ -28,8 +28,7 @@ def icon(self) -> str:
 
     @property
     def pprint(self) -> str:
-        output = f"{self.icon} {self.name} ({self.class_name})@{self.id.emoji()}"
-        return output
+        return f"{self.icon} {self.name} ({self.class_name})@{self.id.emoji()}"
```
Comment (gmuraru) [inline src/syft/core/io/location/specific.py:31]: Q: Why this change?
I was thinking that the ```output = ...``` is there for debugging purposes (if you have ```pdb``` and then you want to check out the message with a simple ```print(output)```.

File: tests/syft/core/node/common/action/function_or_constructor_action_test.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+def test_run_function_or_constructor_action_serde() -> None:
+    alice = sy.VirtualMachine(name="alice")
+    alice_client = alice.get_client()
+
+    path = "torch.Tensor.add"
+    args = (
+        th.tensor([1, 2, 3]).send(alice_client),
+        th.tensor([4, 5, 5]).send(alice_client),
+    )
+    kwargs = {}  # type: ignore
```
Comment (gmuraru) [inline tests/syft/core/node/common/action/function_or_constructor_action_test.py:24]: Q: Why not simply remove this and pass ```kwargs={}```

**Review by gmuraru — APPROVED** [review]

File: src/syft/core/common/serde/deserialize.py [hunk 3]
```diff
@@ -81,8 +81,7 @@ def _deserialize(
         obj_type = type(blob).schema2type  # type: ignore
 
     # uh-oh! Looks like the type doesn't exist. Let's throw an informative error.
-    except KeyError:
-
+    except AttributeError:
         raise TypeError(
             """You tried to deserialize an unsupported type. This can be caused by
             several reasons. Either you are actively writing Syft code and forgot
```
Comment (gmuraru) [inline src/syft/core/common/serde/deserialize.py:87]: Q: Could you also add a space at the end of ```forgot``` :D

**Comment by madhavajay** [comment]
I tried to figure out why that lint error is happening:
https://github.com/OpenMined/PySyft/pull/4564

Feel free to cherry pick or merge from this PR.

**Comment by Jasopaum** [comment]
Merged with #4564
